guides/common/modules/proc_creating-a-user.adoc

```diff
@@ -42,6 +41,10 @@ By default, {ProjectServer} uses the language and timezone settings of the user
 +
 The `--auth-source-id 1` setting means that the user is authenticated internally, you can specify an external authentication source as an alternative.
 Add the `--admin` option to grant administrator privileges to the user.
-Specifying organization IDs is not required, you can modify the user details later using the `update` subcommand.
+Specifying organization IDs is not required.
++
+You can modify the user details later by using the `update` subcommand.
+
+.Additional resources
 
-For more information about user related subcommands, enter `hammer user --help`.
+* For more information about creating user accounts by using Hammer, enter `hammer user create --help`.
```

Comment:
I'm used to using Additional resources for man pages and --help commands from RHEL docs ([a random example](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index#switching-the-system-wide-crypto-policy-to-mode-compatible-with-previous-systems_using-the-system-wide-cryptographic-policies)). I find including it helpful because it gives an immediate hint on where to learn more about the command usage if a user struggles with it.

The original procedure included it within the step. I think moving it to Additional resources is an improvement. But if there is agreement that removing it altogether is the best course of action, I can do it.

Reply:
It is true that we usually use Additional resources for doc links, but I don't think it's a rule. I like the idea of including other helpful resources like man pages or `--help` commands a lot.